charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml

```diff
@@ -97,7 +97,7 @@ spec:
   {{- end }}
 {{- end }}
         ports:
-          - containerPort: {{ .Values.ports.apiPort }}
```

Comment:
Why is this needed?

Reply:
This was a variable under the placement charts which defines the port of the placement service.
In order to be able to access it from the injector chart, I moved it to global.